src/collections/_documentation/performance/performance-glossary.md

```diff
@@ -0,0 +1,170 @@
+  "parent_span_id": "ba99c37bfb30f860",
+  "span_id": "92ce7ea47d5b8a45",
+  "start_timestamp": 1583868931.940406,
+  "same_process_as_parent": true,
+  "description": "updateJiraTask",
+  "tags": { "status": "ok" },
+  "timestamp": 1583868932.060615,
+  "op": "jira",
+  "data": {}
+}
+
+// sql.update - a child span of updateJiraTask
+{
+  "trace_id": "a8233eb845154a2cae4712b28375d08d",
+  "parent_span_id": "92ce7ea47d5b8a45",
+  "span_id": "a88e33dab5b745b9",
+  "start_timestamp": 1583868931.953386,
+  "same_process_as_parent": true,
+  "description": "sql.update - SELECT * FROM jira_table",
+  "tags": { "status": "ok" },
+  "timestamp": 1583868932.03692,
+  "op": "db",
+  "data": {}
+}
+```
+
+### Hierarchy
+
+Each span can be connected to other spans using a parent-child hierarchy. 
+
+For example, a span called `updateJiraTask` would have child spans like `Check permissions`, `Update Task`, `sql.update`, `send.webhooks`.
+
+Each sub-task span would have `parent_span_id` equal to the id of the `updateJiraTask` span.
+
+### Orphan Spans
+
+Orphan spans are spans that lack visible parent spans within Sentry's database. This could imply a bug in the Sentry application. If you're confident about this, please contact support: [support@sentry.io](mailto:support@sentry.io)
+
+Another reason could include that instrumented spans may not make it back to Sentry for processing. As an example, let us consider a web application consisting of:
+
+- Frontend (Single-Page Application)
+- Backend
+- Background Queue
+- Notification Job
+
+Instrumented spans in each component are sent back to Sentry for processing. However, a server in the backend component could suffer from network connectivity loss, and the spans may not have made it back to Sentry. This would indicate a "hole" within this specific trace. The collected spans within the Background Queue would be descendants of a span (for example, a queue service call) within the Backend component, and thus the top-level span of the collective Background Queue spans is referred to as an orphan span.
+
+### Properties
+
+Each span has a unique identifier kept in the `span_id` attribute.
+
+Start time and end time are stored in the `start_timestamp` and `timestamp` attributes, respectively. The format is an RFC 3339 / ISO 8601-compatible string or a numeric value with the number of seconds since the UNIX Epoch.
+
+The name of the operation is stored in the `op` property. Example `op` values are `http`, `sql.query`, `redis.command`.
+
+Additional data about the operation can be stored in the `data` and `tags` attributes.
+
+## Trace
+
+A trace is a collection of transactions that have the same `trace_id` attribute. Traces consist of all transactions generated by the distributed system during the processing of the input data. Traces are not explicitly started or finished. The start and end of a trace are defined by the transactions and spans participating in the trace.
```

Comment:
@markstory @dashed I replaced "spans" with "transactions" for most of this paragraph. Will you double check the technical accuracy?

Reply:
@MimiDumpling double checked. This looks good to me 👍

Reply:
Looks good :clap: